bin/birdhouse

```diff
@@ -115,6 +115,7 @@ Options:
   -a, --component-application-data    Backup application data for a given component (e.g. magpie). This can be specified multiple times. Use '*' to select all components.
   -u, --component-user-data           Backup user data for a given component (e.g. cowbird). This can be specified multiple times. Use '*' to select all components.
   -l, --component-logs-data           Backup logs data for a given component (e.g. thredds). This can be specified multiple times. Use '*' to select all components.
+  -r, --representative-data           Backup representative data for a given component (e.g. stac). This can be specified multiple times. Use '*' to select all components.
```

Comment:
Since this option is only for `stac` and the moment, could we add "Not all components are ready for this mode.  Please double check before enabling this mode"?  Just to clarify user expectations.

Also, what happen if the use really does `-r '*'` but only `stac` is supported?

Reply:
Ah ok, I didn't know that "note" section is there.  All fine then.